app/helpers/cloud_volume_helper/textual_summary.rb

```diff
@@ -36,11 +36,11 @@ def textual_size
   end
 
   def textual_bootable
-    {:label => _('Bootable'), :value => @record.bootable.to_s}
+    {:label => _('Bootable'), :value => _(@record.bootable.to_s)}
```

Comment:
We are going to have to find a way to get these into the translation catalog if they don't already exist.  (Guessing the values here are True/False?)

Reply:
this one is true / false and they already exist on the catalog ✨